### PR TITLE
[IMP] general: integrations outlook plugin add block tiku

### PR DESCRIPTION
**Pull request by tiku-odoo**
Task: 
https://www.odoo.com/web#id=3811146&cids=3&menu_id=4720&action=333&active_id=3835&model=project.task&view_type=form

### CODE REVIEW

**Comment by tiku-odoo** [comment]
@StraubCreative 

Here is another small edit for the Outlook plugin doc, adding a block for accessing the menu in locally installed versions of MS Outlook. 

Thanks, 
Tim :+1:

File: content/applications/general/integrations/mail_plugins/outlook.rst [hunk 1]
```diff
@@ -35,6 +35,12 @@ Next, open the Outlook mailbox, and select any email. After completing this, cli
    :align: center
    :alt: More actions button in Outlook
 
+.. tip::
+   For locally installed versions of Microsoft Outlook, access the :guilabel:`Get Add-ins` menu item
+   while in preview mode (**not** with a message open). First, click on the :guilabel:`...
+   (ellipsis)` icon in the upper right of the previewed message, then scroll down, and click
+   on :guilabel:`Get Add-ins`.
```
Comment (StraubCreative) [inline content/applications/general/integrations/mail_plugins/outlook.rst:42]: line break
```suggestion
   (ellipsis)` icon in the upper right of the previewed message, then scroll down, and click on
   :guilabel:`Get Add-ins`.
```

**Review by StraubCreative — APPROVED** [review]
@tiku-odoo one small change please and can merge, thanks!
..
robodoo delegate=tiku-odoo

**Comment by tiku-odoo** [comment]
@robodoo r+

**Pull request closed without merging**